doc/contributing/coding-conventions.chapter.md

```diff
@@ -196,7 +196,23 @@
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
 
-- Arguments should be listed in the order they are used, with the exception of `lib`, which always goes first.
+- The order of elements in the argument set must follow the convention below:
+
+  01. `lib`;
+  02. `stdenv`;
```

Comment:
I would argue that this should be merged into the builders, if not, other types of stdenv should be mentioned as well, e.g. `clangStdenv`, `stdenvNoCC`

Reply:
"`stdenv` and friends" would be a good name?

Or you are also suggesting to include/lump stdenvs and builders here?

Reply:
> Or you are also suggesting to include/lump stdenvs and builders here?

I would prefer that, yes